Cargo.toml

```diff
@@ -114,6 +114,7 @@ serde_json = "1.0.105"
 serde_with = "3.0.0"
 sha2 = "0.10.7"
 sha3 = "0.10"
+smallvec = "1.13.1"
```

Comment:
iirc `smallvec` is removed on `main`: https://github.com/eqlabs/pathfinder/pull/1895